recipes/magnum/all/conanfile.py

```diff
@@ -433,7 +482,7 @@ def package(self):
                         endif()
                     """.format(target=target, library=library)))
 
-        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, "share", "cmake"))
```

Comment:
```suggestion
        tools.rmdir(os.path.join(self.package_folder, "share", "cmake"))
        tools.rmdir(os.path.join(self.package_folder, "share"))
```

Reply:
```
[HOOK - conan-center.py] post_package(): ERROR: [DEFAULT PACKAGE LAYOUT (KB-H013)] Unknown folder 'share' in the package (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H013) 
ERROR: [HOOK - conan-center.py] post_package(): Some checks failed running the hook, check the output
```

Reply:
Inside `share` there is a folder `magnum` with some files that are distributed together with the library. I can move them to a different place, or modify the hook to accept them. Wdyt @uilianries ?

Reply:
I would suggest `res` folder, as it's managed by Conan. Of course, need to check if does not break the project with some hardcoded expected path.